.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Tells Git to (1) normalize line endings to LF on checkin and 
+# (2) prevent conversion to CRLF when the file is checked out.
+configuration-sample/ods-core.env text eol=lf
+configuration-sample/ods-core.env.sample text eol=lf
```

Comment:
Thanks, this is a good idea! I do not have a Windows machine, but from what I understand, the `text` attribute should be enough so that Git uses `LF` in its storage. `eol=lf` would force this in the working copy, which I'm not sure causes issues in Windows editors?

Reply:
You're absolutely right, using `text` ensures line endings are converted to LF in the repository and that should be enough.